### ISSUE TITLE
The window background color is incorrect on the HighContrastWhite theme

### ISSUE THREAD

**Issue by ssaran046** [microsoft/microsoft-ui-xaml#5706]
**Describe the bug**

The window background color is incorrect on the HighContrastWhite theme.

[HighContrast_Issue.zip](https://github.com/microsoft/microsoft-ui-xaml/files/6998329/HighContrast_Issue.zip)

**Steps to reproduce the bug**

1. Turn on HightContrastTheme on your system and set the theme as "HighContrastWhite"
2. Run the attached sample
3. The window background is black instead of white

**Expected behavior**

The window background color should be white.

**Screenshots**

![image](https://user-images.githubusercontent.com/24865096/129691021-5507dbdd-ff95-486e-b08d-494688ad3dd9.png)

**Version Info**

NuGet package version: [WinUI 3 - Windows App SDK 0.8: 0.8.0]

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
|  | Yes |

| Windows version                    | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2021 Update (19043)            | Yes |
| October 2020 Update (19042)        |    |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

**Comment by chrisglein** [comment]
Theming used the CoreWindow to hook these sorts of events, but in Desktop WinUI3 that's not a thing. So this is likely not yet implemented in WinUI3.
